packages/db/src/services/DatabaseService.ts

```
@@ -1,39 +1,370 @@
+    } catch (error: unknown) {
+      console.error("Error updating driver info: " + (error as Error).message);
+      throw new Error((error as Error).message);
     }
   }
 
   async close(): Promise<void> {
-    if (this.dataSource.isInitialized) {
-      await this.dataSource.destroy();
+    if (this.isConnected) {
+      await AppDataSource.destroy();
       console.log("Database connection closed");
```

ESLint

[error] Unexpected console statement.

(no-console)